client/lib/abtest/active-tests.js

```diff
@@ -65,6 +71,18 @@ module.exports = {
 		defaultVariation: 'drake',
 		allowExistingUsers: true,
 	},
+
+	addCustomDomainOnSiteSettingsPage: {
+		datestamp: '20160425',
+		variations: {
+			byAddCustomDomainButton: 50,
```

Comment:
Since we are now showing the add custom domain button regardless of this test, I think this variant should be renamed `default` or `control`. I'm also not sure why the variation names are prefixed with "by".